boards/adafruit/feather_esp32s2/adafruit_feather_tft_reverse_esp32s2_defconfig

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_CONSOLE=y
```

Comment:
For the TFT and Reverse boards, I do not see where this entries are needed CONSOLE, SERIAL, UART_CONSOLE...) Is it required?

Reply:
Actually the same as above, I actually did the same as the other EPS32-S2 Boards like the ESP32-S2-DevKitC, ESP32-S2-Saola, etc.

If you think it should be removed, I will remove it.